pkgs/applications/graphics/unigine-valley/default.nix

```diff
@@ -1,115 +1,138 @@
-{ lib, stdenv, fetchurl
+{ lib
+, stdenv
+, fetchurl
 
-# Build-time dependencies
+  # Build-time dependencies
 , makeWrapper
 , file
 
-# Runtime dependencies
+  # Runtime dependencies
 , fontconfig
 , freetype
 , libX11
 , libXext
 , libXinerama
 , libXrandr
 , libXrender
-, libGL
-, openal}:
+, libglvnd
+, openal
+, makeDesktopItem
+, imagemagick
```

Comment:
nitpick: makeDesktopItem and imagemagick are build-time dependencies

Reply:
thas why I am against adding such section comments.